samples/KvsVideoOnlyStreamingSample.c

```diff
@@ -82,7 +82,7 @@ INT32 main(INT32 argc, CHAR* argv[])
     }
 
     MEMSET(frameFilePath, 0x00, MAX_PATH_LEN + 1);
-    if (argc < 5) {
+    if (argc < 6) {
```

Comment:
Can we add this property to the usage print statement (line 74)?

Reply:
Done, also updated the README